src/main/java/carpetextra/mixins/DispenserBehaviorCarpetsMixin.java

```diff
@@ -9,10 +9,9 @@
 import org.spongepowered.asm.mixin.injection.Inject;
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
 
-@Mixin(targets = "net/minecraft/block/dispenser/DispenserBehavior$4")
+@Mixin(targets = "net/minecraft/block/dispenser/DispenserBehavior$13")
```

Comment:
not sure if defect, but placing carpets with carpet carpets rule on does not issue the result of the command (e.g., pink carpet should print the spawn conditions). I'll review prior versions behavior and mark resolved if not needed.

Reply:
This is not a new defect (if it is even a defect).